tray/actions.go

```diff
@@ -75,6 +77,17 @@ func (ti *Instance) logout(persistToken bool) bool {
 	}
 }
 
+func (ti *Instance) notifyServiceExpired(url string, trustedPassURL string, message string) {
+	resp, err := ti.client.TokenInfo(context.Background(), &pb.Empty{})
+	isTokenDataValid := resp.TrustedPassToken != "" && resp.TrustedPassOwnerId != ""
+
+	link := url
+	if err == nil && isTokenDataValid {
+		link = fmt.Sprintf(trustedPassURL, resp.TrustedPassToken, resp.TrustedPassOwnerId)
+	}
+	ti.notify(message, link)
```

Comment:
wouldn't make sense in this case to force show notification, in case the user has them disabled?